examples/log-extension/src/__tests__/unit/providers/log-action.provider.unit.ts

```diff
@@ -3,23 +3,22 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {sinon} from '@loopback/testlab';
 import {Request} from '@loopback/rest';
+import {sinon} from '@loopback/testlab';
+import chalk from 'chalk';
 import {
+  HighResTime,
+  log,
   LogActionProvider,
   LogFn,
   LogWriterFn,
-  log,
   LOG_LEVEL,
-  HighResTime,
 } from '../../..';
-import chalk from 'chalk';
-
-import {createLogSpy, restoreLogSpy, createConsoleStub} from '../../log-spy';
 import {logToMemory} from '../../in-memory-logger';
+import {createConsoleStub, createLogSpy, restoreLogSpy} from '../../log-spy';
 
 describe('LogActionProvider with in-memory logger', () => {
-  let spy: sinon.SinonSpy;
+  let spy: sinon.SinonSpy<[(string | undefined)?], void>;
```

Comment:
There was an error showing up while running tests. It needed explicit definition of type for the generic param. So I added it. Rest of the file changes are just reordering of imports done by vscode extension TSHero. I hope that's fine.